src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/KeyRow.vue

```diff
@@ -115,17 +126,11 @@ export default Vue.extend({
 
 <style lang="scss" scoped>
 @import '~styles/tables';
-.icon {
-  margin-left: 18px;
-  margin-right: 20px;
-}
-
-.clickable {
-  cursor: pointer;
-}
 
-.no-border {
-  border-bottom-width: 0 !important;
+.xrd-table {
```

Comment:
Instead of this xrd-table customising use pl-10 etc. from https://vuetifyjs.com/en/styles/spacing/

Reply:
Fixed